sunpy/net/jsoc/jsoc.py

```diff
@@ -410,7 +410,7 @@ def search_metadata(self, *query, **kwargs):
 
         return res
 
-    def request_data(self, jsoc_response, **kwargs):
+    def request_data(self, jsoc_response, download_tar=False, **kwargs):
```

Comment:
Would it not be more flexible to allow the user to specify `method=` and `protocol=` explicitly here?

Reply:
True, but do we want to support all methods/protocols combinations? If so, i can create changes for that. I guess, supporting ftp is ok, but not so sure regarding jpeg/mpeg/mp4 because of processing and additional options you have to select in JSOC export (colormap, scaling, img size).

Reply:
I think you are correct that we don't really want to support all the possible options, but I think building the API in a way which is extensible into the future so we can add them if we want is a better choice.

I would suggest you allow things which work with minimal modification and disallow everything else, and then there is simple (non-API changing) path forward for adding them in the future.

Reply:
Ok, i agree. I will re-work it.